scripts/examples/50-OpenMV-Boards/60-Shields/60-LCD-Shield/lcd_shield.py

```diff
@@ -0,0 +1,26 @@
+# This work is licensed under the MIT license.
+# Copyright (c) 2013-2025 OpenMV LLC. All rights reserved.
+# https://github.com/openmv/openmv/blob/master/LICENSE
+#
+# LCD Shield Example
+#
+# Note: To run this example you will need a LCD Shield for your OpenMV Cam.
+#
+# The LCD shield allows you to view your OpenMV Cam's frame buffer on the go.
+
+import sensor
+import time
+import display
+import image
+
+sensor.reset()
+sensor.set_pixformat(sensor.RGB565)
+sensor.set_framesize(sensor.LCD)
+
+lcd = display.SPIDisplay(vflip=True, hmirror=True)
```

Comment:
Please revert the notes about the backlight controller.  Also, would `portrait` arg work here?

Reply:
Reverted, regarding portrait, no, that's a rotate 90, so it would be like transpose + hmirror at the same time. vflip+hmirror is a rotation of 180.